botogram/defaults.py

```diff
@@ -58,7 +58,9 @@ def help_command(self, bot, chat, args):
                                  name=args[0]),
                            bot._("Use /help to get a list of the commands.")]
         else:
-            message = self._help_generic_message(bot, commands)
+            sorted_commands = {cmd.name: cmd for cmd in
```

Comment:
Maybe you can change the help message code to deal with an iterator of `Command`s, instead of converting it to a dict: this way you preserve ordering.